src/components/select/select.js

```diff
@@ -135,6 +137,18 @@ angular.module('material.components.select', [
  *     </md-select>
  *   </md-input-container>
  * </hljs>
+ * Multiselect with a custom separator
+ *
+ * When a developer needs to add a seperator with multi select, adding to the attribute `selection-separator-char` or
+ `selection-separator-spacing` with options. The example below demonstrate a new line for after each selected item.
```

Comment:
"The example below demonstrates how to use a new line after each selected item."